namespace/other-javax-references.adoc

```diff
@@ -30,4 +30,24 @@ TODO Help
 
 TODO Help
 
-## TODO Help list every spec
\ No newline at end of file
+## TODO Help list every spec
+
+## JTA
+The following classes are referenced from `javax.sql.XAConnection`which is still part of JavaSE
```

Comment:
I think the javax.sql.XAConnection reference might not be the best one to use as JTA is not the source of the imports? As it reads effectively it is saying that a JavaSE class references other JavaSE classes? That being said I did find there:

```
$ find . -name \*.java -exec grep "import javax" {} \;
import javax.transaction.xa.XAResource;
import javax.enterprise.util.Nonbinding;
import javax.interceptor.InterceptorBinding;
import javax.enterprise.context.NormalScope;
```

Reply:
indeed, the whole javax.transaction.xa package is still part of SE. But all the rest beneath javax.transaction is EE.

Reply:
I should have mentioned when I did the find I was in git@github.com:eclipse-ee4j/jta-api.git

Reply:
What I intended to say (please reword as necessary): the `javax.transaction.*` package is listed as being moved to `jakarta.transaction.*`. But this is indeed not fully true as `javax.transaction.xa.*` is part of JavaSE and not subject of being moved.

The whole `javax.transaction.xa.*` package is not part of the source code in https://github.com/eclipse-ee4j/jta-api and this is good. 
But still we need to keep this in mind when it comes to replacing imports in projects later.
Makes sense?

Reply:
It does make sense with the new wording, I think the PR could be updated to reflect that thanks!

Reply:
@tomjenkinson, @struberg, can one of you hit the "Resolve conversation" button so I can merge this PR?
Thanks.

Reply:
@bshannon no problem, resolving

Reply:
or indeed, "problem" - it seems I don't have the permission to do that on this repo sorry. But the conversation is resolved from my point of view @bshannon